frontend/src/components/cards/miniCard.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .MiniCardImageContainer:hover {
-    border-color: var(--global--active--highlighting-color);
+    border-color: #ae56ff;
```

Comment:
Can't we have a variable here? would probably be better for readability and such ._.